utils/bazel/llvm-project-overlay/mlir/BUILD.bazel

```diff
@@ -5277,6 +5277,7 @@ cc_library(
         ":LLVMCommonConversion",
         ":LLVMDialect",
         ":MemRefDialect",
+        ":MLIRArithDialect",
```

Comment:
I think we need a similar change in `mlir/lib/Conversion/NVGPUToNVVM/CMakeLists.txt`